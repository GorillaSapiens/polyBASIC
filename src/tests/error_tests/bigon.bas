on 5 goto 10,20,30,40
