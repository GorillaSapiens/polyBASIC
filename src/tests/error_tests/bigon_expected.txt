SOURCE 1:4, INDEX 5 OUT OF RANGE
